src/components/atom_word_cards_frame_surfing_button/index.tsx

```diff
@@ -18,6 +18,9 @@ const WordCardsFrameSurfingButton: FC = () => {
       },
     [onSemesterClick],
   )
+  const semesters = useRecoilValue(semestersState)
+  if (semesters === undefined || semesters.length < 3) return null
```

Comment:
Can you disable the icon instead of returning `null`?

https://github.com/ajktown/wordnote/blob/main/src/atoms/StyledCloudRefresher.tsx#L78

Reply:
handled